docs/tutorials/orc.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<p>I think this feature processing step can be modified to use the keras pre-processing layers.</p><p>For ex: - https://www.tensorflow.org/tutorials/structured_data/preprocessing_layers#categorical_columns</p><p>https://www.tensorflow.org/tutorials/structured_data/preprocessing_layers#numeric_columns</p><p> </p><p>Please modify the next cells as required.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/io/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>keras preprocessing is experimental. It is better to use non experimental features in tutorial so it can be long lasting. We can update it later?</p>

Reply:
<p>okay sure.</p>

Reply:
just FYI, even though they are experimental, they are quite stable and are widely used.